feeds/pypi/main.go

```diff
@@ -58,32 +77,47 @@ func fetchPackages() ([]*Package, error) {
 	return rssResponse.Packages, nil
 }
 
-// PubSubMessage is the payload of a Pub/Sub event.
-type PubSubMessage struct {
-	Data []byte `json:"data"`
-}
-
 // Poll receives a message from Cloud Pub/Sub. Ideally, this will be from a
 // Cloud Scheduler trigger every `delta`.
-func Poll(ctx context.Context, m PubSubMessage) error {
+func Poll(ctx context.Context, t *pubsub.Topic) error {
 	packages, err := fetchPackages()
 	if err != nil {
 		return err
 	}
 	cutoff := time.Now().UTC().Add(-delta)
 	for _, pkg := range packages {
+		log.Println(pkg.Name(), pkg.Version())
 		if pkg.ModifiedDate.Before(cutoff) {
 			continue
 		}
-		// TODO: publish the package up to a cloud pub/sub for processing
-		packages = append(packages, pkg)
+		msg := library.Package{
+			Name:    pkg.Name(),
+			Version: pkg.Version(),
+			Type:    processor.TypePyPI,
+		}
+		b, err := json.Marshal(msg)
+		if err != nil {
+			return err
+		}
+		if err := t.Send(ctx, &pubsub.Message{
+			Body: b,
+		}); err != nil {
+			return err
+		}
+		log.Println("Sent")
```

Comment:
Small nit: should we consolidate the first log message and this one so that we have a single message we can review that says "sent <package> <version>"? I feel like that'll make debugging easier. Totally fine if we want to keep the first message around so that we can see both what packages we've seen, and which we've sent upstream.

Reply:
Whoops, meant to drop these. I added them when debugging and got confused.